src/packages.builds

```diff
@@ -3,9 +3,10 @@
 
   <PropertyGroup Condition="'$(DotNetBuildFromSource)' == 'true'">
     <AdditionalBuildConfigurations>$(AdditionalBuildConfigurations);package-$(ConfigurationGroup)</AdditionalBuildConfigurations>
+    <BuildAllPackages>false</BuildAllPackages>
```

Comment:
Why is that change necessary?

Reply:
In the servicing releases, we just publishes the packages that we changed in that particular release cycle. I think that should be the reason.
cc @wtgodbe just to verify as I added most of the stuff from https://github.com/wtgodbe/corefx/commit/6b2de6b88c94fad7a844f4d05b0d18090eb32fa5

Reply:
Should add something similar to https://github.com/wtgodbe/corefx/commit/6b2de6b88c94fad7a844f4d05b0d18090eb32fa5#diff-cb73a137838f73872d476fa5146bfedeR18-R27 as well to ensure that we always build the runtime package + the package baseline, and so we can add specific packages to be built for servicing.